pkg/network/protocols/postgres/model_linux_test.go

```diff
@@ -0,0 +1,80 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux_bpf
+
+package postgres
+
+import (
+	"testing"
+
+	"github.com/DataDog/go-sqllexer"
+	"github.com/stretchr/testify/require"
+)
+
+func TestExtractTableFunction(t *testing.T) {
+	tests := []struct {
+		name       string
+		query      string
+		tablesName string
+	}{
+		{
+			name:       "single table name",
+			query:      `DROP TABLE IF EXISTS test1`,
+			tablesName: "test1",
+		},
+		{
+			name:       "multiple table names",
+			query:      `DROP TABLE IF EXISTS test1;DROP TABLE IF EXISTS test2;`,
+			tablesName: "test1,test2",
+		},
```

Comment:
The case is problematic as - 
1. You can have different operations (drop and select for example)
2. there are 2 queries here, and not a single query. The first query operates on `test1`, and the second operates on `test2`. So having a table name `test1,test2` is not accuracte
3. We should support (in the future) multiple commands, but currently we're not up to do so

Reply:
I understand what you are saying those are good points.
I have removed the test for this senario and changed the join to include only the first table which it find, which supposed to be the only case we are currently support. As well left a comment to clear why I am doing so.